### DISCUSSION TITLE
Pause stops playback

### DISCUSSION THREAD

**Discussion by mterekhov** [gerbera/gerbera#3024] [Q&A]
Hello everyone! I have a problem when watching video on my SamsungTV(qe55q70bauxcxe) - if I press pause and then play the playback process stops and TV says that connection with server is lost. Is there any advices or workaround to get pause work properly?

**Comment by KarlStraussberger** [comment]
The first question always is: what is your gerbera version?

If you are running something recent (2.0; 1.9.2 only if the issue is really obvious), the next is: can you provide logs created when running with `--debug`?
